test/Microsoft.AspNetCore.Server.KestrelTests/EngineTests.cs

```diff
@@ -301,11 +301,14 @@ public void ConnectionCanReadAndWrite(TestServiceContext testContext)
             {
                 using (var connection = new TestConnection(server.Port))
                 {
-                    await connection.Send(
```

Comment:
Just checking if this can be reverted back to `Send`; would prefer to keep the test the same (other than correcting the input)